### ISSUE TITLE
Content under transparent appbar

### ISSUE THREAD

**Issue by awaik** [BilalShahid13/PersistentBottomNavBar#108]
Hi, thank you for the great package!
Is it possible to achieve this kind of effect with your package?

<img width="345" alt="Screenshot 2020-11-15 at 12 36 14" src="https://user-images.githubusercontent.com/5330807/99181403-55276480-273f-11eb-8a2f-0ce4693fcddd.png">

**Comment by BilalShahid13** [comment]
Yes, it's possible. Use the `opacity` and optionally `filter` argument in `PersistentBottomNavBarItem`.
